src/com/zone5cloud/retrofit/core/apis/ThirdPartyTokenAPI.java

```diff
@@ -48,4 +46,38 @@
     @DELETE(Users.DEREGISTER_DEVICE_THIRD_PARTY_CONNECTION)
     Observable<Response<Void>> deregisterDeviceWithThirdParty(@Path("token") String token);
 
+    /**
+     * The name of the service that will be connected.
+     * @param service The name of the service that will be connected.
+     * @param emptyObject an empty body
+     * @return On a successful request returns  {@link ConnectionInitResponse} containing
+     * the endpoint returns an OAuth URL allowing you to log into and authorize the third-party service.
+     */
+    @POST(ThirdParty.INIT_CONNECTION_PAIRING)
+    Observable<Response<ConnectionInitResponse>> initConnectionPairing(@Path("service") UserConnectionsType connectionType, @Body Object emptyObject);
+
```

Comment:
The path variable defined in ThirdParty.INIT_CONNECTION_PAIRING is "connectionType", not "service"